neasqc_wp61/6_Beta_Neighbours.sh

```diff
@@ -3,26 +3,21 @@
 echo 'This script classifies examples using beta neighbours model.'
 
 
-while getopts l:t:v:k:r:o: flag
+while getopts "t:v:k:o:" flag
 do
     case "${flag}" in
-        l) labels=${OPTARG};;
         t) train=${OPTARG};;
         v) test=${OPTARG};;
-        k) k=${OPTARG};;
+        k) IFS=' ' read -r -a k <<< "${OPTARG}";;
         o) output=${OPTARG};;
     esac
 done
 
-echo "labels: $labels";
 echo "train: $train";
 echo "test: $test";
-echo "k: $k";
-echo "runs: $runs";
+echo "K values for KNN algorithm: ${k[*]}";
 echo "output: $output";
 
+echo "running beta_1"
 
-
-
-echo "running beta neighbours"
-python3.10 ./data/data_processing/use_beta_neighbours.py -l ${labels} -tr ${train} -te ${test} -k ${k} -o ${outfile}
\ No newline at end of file
+python3.10 ./data/data_processing/use_beta_1.py -tr ${train} -te ${test} -k ${k[*]} -o ${output}
```

Comment:
/!\ Here in the set_up_exp PR, the beta folder + beta files are renamed. It's not the case for the new general_quantum_distance PR.

Reply:
\+ Need to add the PCA arg